src/org/traccar/Context.java

```diff
@@ -429,6 +395,59 @@ public static void init(String[] arguments) throws Exception {
 
     }
 
+    private static void initLocationResolutionSystem() {
+
+        String type = config.getString("geolocation.type", "mozilla");
+        String url = config.getString("geolocation.url");
+        String key = config.getString("geolocation.key");
+
+        switch (type) {
+            case "google":
+                geolocationProvider = new GoogleGeolocationProvider(key);
+                break;
+            case "opencellid":
+                geolocationProvider = new OpenCellIdGeolocationProvider(key);
+                break;
+            case "unwired":
+                geolocationProvider = new UnwiredGeolocationProvider(url, key);
+                break;
+            default:
+                geolocationProvider = new MozillaGeolocationProvider(key);
+                break;
+        }
+    }
+
+    private static void initEventsSubsystem() {
```

Comment:
Rename to `initEventsSystem` or `initEventsModule` to be consistent with other method.

Reply:
done